config/locales/en.yml

```diff
@@ -193,7 +193,8 @@ en:
           attributes:
             group_id:
               blank: Group must exist
-              taken: The group has already been shared with this namespace
+              taken: has already been shared with this namespace
```

Comment:
Can you add these two translations to the fr.yml file as well

Reply:
fixed in https://github.com/phac-nml/irida-next/pull/649/commits/123eb53b9766548a17ea4c984486578bf393d500